jstl/src/main/resources/com/sun/ts/tests/jstl/compat/onedotzero/positiveFormatLocalizationContextI18NTestJava20Plus.jsp

```diff
@@ -0,0 +1,49 @@
+<%--
+
+    Copyright (c) 2003 Contributors to the Eclipse Foundation
+
```

Comment:
There are files in https://github.com/jakartaee/platform-tck/pull/1320/files#diff-4254e4814daa576f775cd40e835a7e0eca3da9be2b6bd679fcc13be9901fdb94 where I added a new copyright:

`Copyright (c) 2024 Contributors to the Eclipse Foundation`

We should preserve that here since we're just copying that code to a different branch.

Reply:
I won't comment on each file but my comment applies to more than just this file.

Reply:
I corrected copyrights of the new files added.

Reply:
@alwin-joseph shouldn't the Oracle copyright be preserved? Then just the new copyright holder added to the header? https://www.eclipse.org/projects/handbook/#legaldoc-faq. See https://github.com/jakartaee/platform-tck/pull/1320/files#diff-4254e4814daa576f775cd40e835a7e0eca3da9be2b6bd679fcc13be9901fdb94R4 for an example.

Reply:
I created a new PR here with some cleanup :https://github.com/jakartaee/platform-tck/pull/1345